examples/scripts/ppo-sentiment-t5-small.py

```diff
@@ -57,7 +57,7 @@
 # Below is an example function to build the dataset. In our case, we use the IMDB dataset
 # from the `datasets` library. One should customize this function to train the model on
 # its own dataset.
-def build_dataset(tokenizer, dataset_name="imdb", input_min_text_length=2, input_max_text_length=8):
+def build_imdb_dataset(tokenizer, input_min_text_length=2, input_max_text_length=8):
     # load imdb with datasets
     ds = load_dataset(dataset_name, split='train')
```

Comment:
then also this needs to change, sorry didn't see it
```suggestion
    ds = load_dataset("imdb", split='train')
```